apps/central/src/components/SchemaEditModal.vue

```diff
@@ -92,10 +103,15 @@ export default {
       this.success = null;
       request(
         this.endpoint,
-        `mutation updateSchema($name:String, $description:String){updateSchema(name:$name, description: $description){message}}`,
+        `mutation updateSchema($name:String, $description:String, $isChangelogEnabled: Boolean) {
```

Comment:
sorry to be such a big pain in the 4$$ but this morning I woke up wondering why we didn't user 'settings' in case?